lib/checkfunctions.cpp

```diff
@@ -188,7 +188,7 @@ void CheckFunctions::checkMathFunctions()
         for (const Token* tok = scope->classStart->next(); tok != scope->classEnd; tok = tok->next()) {
             if (tok->varId())
                 continue;
-            if (printWarnings) {
+            if (printWarnings && Token::Match(tok, "%name% ( %any% [,)]")) {
```

Comment:
@PKEuS The pattern you propose would allow `func()` which is not allowed by any of the patterns applied later.